react/src/components/InviteFolderSettingModal.tsx

```diff
@@ -274,22 +275,32 @@
                             handlePermission(record.shared_to.uuid, perm);
                           }}
                         />
-                        <Tooltip
-                          title={t('data.folders.KickOut')}
-                          placement="right"
+                        <Popconfirm
+                          title={t('data.folders.KickOutConfirm', {
+                            email: record.shared_to.email,
+                          })}
+                          okText={t('button.Confirm')}
+                          okButtonProps={{
+                            danger: true,
+                          }}
+                          onConfirm={() => {
+                            handlePermission(record.shared_to.uuid);
+                          }}
                         >
-                          <Button
-                            type="text"
-                            style={{
-                              color: token.colorError,
-                              background: token.colorErrorBg,
-                            }}
-                            icon={<TrashBinIcon />}
-                            onClick={() => {
-                              handlePermission(record.shared_to.uuid);
-                            }}
-                          />
-                        </Tooltip>
+                          <Tooltip
+                            title={t('data.folders.KickOut')}
+                            placement="right"
+                          >
```

Comment:
Currently, the tooltip does not disappear when the button is clicked, so it remains on the screen like Popconfirm. 

What about setting the tooltip to disappear on click? 
- Wrapping a Tooltip around the top of a Popconfirm
- Set the Tooltip placement property to 'top'

Reply:
Thank you for suggestion. It's better to control the `open` status using `useState`.